security/mkcert/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+
+github.setup        FiloSottile mkcert 1.1.0 v
+
+platforms           darwin
+categories          security devel
+license             permissive
+installs_libs       no
```

Comment:
What's `installs_libs` here for?

Reply:
`installs_libs no` indicates that the port installs no libraries. Therefore any ports that have this port as a dependency don't need to worry about architecture matching.

Reply:
>Thus architecture matching still matters.

I don't think so, Frank.